product-service-new/src/handlers/getProductsById.ts

```diff
@@ -0,0 +1,38 @@
+import books from './books.json'
+import { APIGatewayProxyHandler } from 'aws-lambda'
+
+export const getProductsById: APIGatewayProxyHandler = async (event) => {
+	try{
```

Comment:
The same as `await` usage: where do you think you can get an error? 1 variant i have seen to get books with `Promise.resolve` method. But you didn't choose this way.

Reply:
all was according to the task